examples/macos/text_edit_item.py

```diff
@@ -31,14 +31,23 @@
 
 """Run some automations to test things"""
 from __future__ import unicode_literals
-from __future__ import print_function
-import sys
-sys.path.append('/Users/daria/pywinauto')
-
-from pywinauto import application
-from pywinauto import macos 
+from __future__ import print_function
 
-app = application.Application()
-app.start('TextEdit')
-app.kill()
+import sys
 
+sys.path.append(".")
+
+from pywinauto.macos.application import Application
+
+app = Application()
+app.start('TextEdit')
+text_edit = app.Window_().TextEdit
+main_win_wrapper = text_edit.wrapper_object().set_focus()

```

Comment:
`main_win_wrapper` sounds like incorrect name. Maybe `text_edit_wrapper`? And maybe use it everywhere below?

Reply:
Change name to text_edit_wrapper. But if I use text_edit_wrapper below (i.e text_edit_wrapper.type_keys('some text') I get error "NoneType object has no attribute 'type_keys'". So, did I put type_keys to wrong place or this behavios is correct?

Reply:
How renaming can change the behavior??? `NoneType` means that returned value is `None`. Of course this object doesn't contain any attribute.

Reply:
I think I misunderstand. I agree main_win_wrapper sounds like incorrect name. And name was changed to text_edit_wrapper. But I don't understand what 'maybe use it everywhere below' means. I thought it meant to use text_edit_wrapper everywhere below. Please correct me if I think wrong